proc_maps32_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aix darwin dragonfly freebsd linux netbsd openbsd solaris
+// +build 386 arm mips mipsle
```

Comment:
I'm wondering if this line should be removed since these tests should work on both 32 bit and 64 bit platforms right?

Reply:
Yes, the 32 bit tests would technically work on 64 bit, however the original test did not compare against the 32 bit fixtures when running on 64 bit, due to the machine word size detection if-statement. I thought it best to preserve that behavior.

Reply:
Ok, makes sense, in that case I'm fine with this as long as @discordianfish and @SuperQ have no objections.